site/src/pages/components/accordion.explainer.mdx

```diff
@@ -234,6 +235,18 @@ See https://github.com/whatwg/html/issues/8864 .
 is one possible interactive presentation
 of markup that is semantically pairs of headings and sections.)
 
+### Accordions with exactly one item open
+
+Some accordions (such as the one on the Pixel product page above)
+always have exactly one item open,
+and do not allow zero items to be open.
+(This could be called an exact-exclusive accordion.)
+This is a somewhat more complicated widget,
+because the open/close control diappears from the item that is open.
```

Comment:
In my look at that page, the "down arrow" widget is still there, it just doesn't do anything. This text is fine, just pointing it out.

Reply:
On the pixel product page, it disappears... but on the FAST design system demo page, it indeed becomes disabled.  I'll revise this to say "or becomes disabled".